disnake/ext/commands/interaction_bot_base.py

```diff
@@ -883,7 +883,7 @@ async def _sync_application_commands(self) -> None:
             msg = "This method is only usable in disnake.Client subclasses"
             raise NotImplementedError(msg)
 
-        if not self._command_sync_flags._sync_enabled or self._is_closed or self.loop.is_closed():
+        if not self._command_sync_flags._sync_enabled or self._is_closed:
```

Comment:
Is this important? I can't remember why i removed this. I certainly wouldn't expect the (global) loop to be closed while the bot is running